packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -71,16 +99,18 @@ export default class QuanticCategoryFacet extends LightningElement {
   get values() {
     return this.state.values;
   }
-
-  get parents() {
-    return this.state.parents;
+  get nonActiveParents() {
+    return this.state.parents?.slice(0, -1);
+  }
```

Comment:
I think some newlines between the functions will be more consistent with what we already have. This applies for the rest of this file as well :)